app/views/admin/image/_image_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= form.fields_for field do |image_form| %>
+  <div class="row">
+    <div class="col-lg-2 col-md-12 field-key">
+      <%= image_form.label :file, field.to_s.capitalize %>
+    </div>
+    <div class="col-lg-10 col-md-12 field-value">
+      <%= image_form.file_field :file, accept: 'image/png,image/gif,image/jpeg'%>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-lg-2 col-md-12 field-key">
+      <%= image_form.label :_destroy, "Delete #{field.to_s.capitalize}" %>
+    </div>
+    <div class="col-lg-10 col-md-12 field-value">
+      <%= image_form.check_box :_destroy %>
```

Comment:
Probably can also add a case to rescue for deleting an image that hasn't been saved... because when I tried attaching an image file (new image) and mark this checkbox as selected, an error occurs:
<img width="1104" alt="Screen Shot 2019-07-16 at 02 27 52" src="https://user-images.githubusercontent.com/28918289/61239539-b0270100-a771-11e9-83da-4c417f277802.png">

Reply:
Why would admin do that??? :v

Anyway, I have done validation on gallery photos so it should works fine for now. Currently, I still see a lot of things need improvement in the validation and I think I had better done the rest in a separate PR.

Reply:
Who knows :)), not really important but ya better prepare for worst case scenario haha

Reply:
Okok, nice, thankss